roles/create_certification_project/tasks/main.yml

```
@@ -7,6 +7,10 @@
     fail_msg: "The parameter cert_item is required to create cert project"
     success_msg: "The parameter {{ cert_item }} is present"
 
+- name: Reset the project id to handle multiple CNF
+  ansible.builtin.set_facts: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)